packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -50,6 +51,24 @@ export const App = ({ flowQueryParams, navigatorLanguages }: AppProps) => {
     Metrics.init(metricsEnabled, flowQueryParams);
   }, [metricsEnabled, flowQueryParams]);
 
+  useEffect(() => {
+    if (!loading) {
+      // For settings app, we only enable Sentry once we know the user's metrics
```

Comment:
Thanks for the comment!

Reply:
Will calling `sentryMetrics.captureException` in our `AppErrorBoundary` cause any new errors if `.disable` is called first?

Also, do we need to do this in payments as well (very possible we've already got a ticket for it, this is just a drive-by review heh)?

Reply:
@LZoog calling `captureException` won't do anything after Sentry has been disabled. We might need to add it to payments server as well but tbd.